tests/data.tf

```diff
@@ -0,0 +1,133 @@
+data "aws_partition" "current" {}
+
+data "http" "ip" {
+  # Get local ip for security group ingress
+  url = "http://ipv4.icanhazip.com"
+}
+
+# get the default vpc
+data "aws_vpc" "this" {
+  default = "true"
+}
+
+# get the list of subnets in the default vpc
+data "aws_subnet_ids" "public" {
+  vpc_id = "${data.aws_vpc.this.id}"
+}
+
+data "null_data_source" "start_time" {
```

Comment:
This is no longer used, might as well remove it.

Reply:
removed